p2p_wallet/UIKit/Views/WalletCells/WalletCell.swift

```diff
@@ -31,6 +38,12 @@ class WalletCell: BaseCollectionViewCell {
         coinLogoImageView.tokenIcon.cancelPreviousTask()
         coinLogoImageView.tokenIcon.image = nil
     }
+
+    func setIsSelected(isSelected: Bool) {
+        coinLogoImageView.alpha = isSelected ? 0.2 : 1
+        coinCheckMark.isHidden = !isSelected
+        contentView.backgroundColor = isSelected ? .h5887ff.withAlphaComponent(0.1) : .white
```

Comment:
avoid using .white, use clear instead, unless the ui in darkmode will be broken
![simulator_screenshot_4EC57845-66AE-4F8A-B5D5-F614AA7DB8CE](https://user-images.githubusercontent.com/6975538/146171361-3e7b3fbf-07b6-418f-b13e-797d891b3739.png)
The same for NavigationBar